pom.xml

```diff
@@ -253,9 +257,14 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>pipeline-stage-step</artifactId>
-            <version>2.3</version>
+            <version>${pipeline-stage-step.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>pipeline-stage-step</artifactId>
+            <version>${pipeline-stage-step.version}</version>
+        </dependency>
```

Comment:
We don't want to add this as a non-test scope dependency. I would check the class name in `DSL` to avoid this like I mentioned separately:

```suggestion
```

Reply:
removed in 8bdc019304535bead1f991374898b23c12ba5513